release/xgboost_tests/cluster_cpu_small.yaml

```diff
@@ -16,20 +17,28 @@ provider:
     availability_zone: us-west-2a
     cache_stopped_nodes: false
 
+available_node_types:
+    cpu_4_ondemand:
+        node_config:
+            InstanceType: m5.xlarge
+        resources: {"CPU": 4}
+        min_workers: 3
+        max_workers: 3
+
 auth:
     ssh_user: ubuntu
 
-head_node:
-    # 64 CPUs
-    InstanceType: m5.xlarge
+head_node_type: cpu_4_ondemand
+worker_default_node_type: cpu_4_ondemand
 
-worker_nodes:
-    # 64 CPUs
-    InstanceType: m5.xlarge
+file_mounts: {
+    "~/release-automation-xgboost_tests": "."
+}
 
 setup_commands:
     - pip install pytest xgboost_ray
     - sudo mkdir -p /data || true
     - sudo chown ray:1000 /data || true
     - rm -rf /data/classification.parquet || true
+    - cp -R /tmp/ray_tmp_mount/release-automation-xgboost_tests ~/release-automation-xgboost_tests || echo "Copy failed"
```

Comment:
Needs to `sudo chown ray /` because the fault tolerance tests try to write to root.

Reply:
I can't find that - did you get an error message for that?

Reply:
Yeah there's a permission denied when the test tries to write the output.